output/mapping/F23_2014.csv

```diff
@@ -128,11 +128,11 @@ xpath,label-key,index,comment,guidance
 /AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_ESTIMATED_TOTAL/@CURRENCY,currency,,,"For the *Initial estimated total value of the contract/lot*, discard as above. For the *Total value of the contract/lot*, map to the contract's `.value.currency`."
 /AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_TOTAL,value_total_final_concess,,,Map to the contract's `.value.amount`
 /AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_TOTAL/@CURRENCY,currency,,,/AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_ESTIMATED_TOTAL/@CURRENCY
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE,concess_fees_prices,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE/@CURRENCY,,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_PRICE_PAYMENT,concess_payments,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_PRICE_PAYMENT/@CURRENCY,,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/INFO_ADD_VALUE,concess_other_details,,,
+/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE,concess_fees_prices,,,Map to `award.concession.revenueFromFees.amount`.
+/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE/@CURRENCY,,,,Map to `award.concession.revenueFromFees.currency`.
+/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_PRICE_PAYMENT,concess_payments,,,Map to `award.concession.revenueFromBuyer.amount`.
```

Comment:
I'm not sure that "Prizes, payments or other financial advantages provided by the contracting authority/entity" means the same thing as "revenue from buyer".

@duncandewhurst might have suggestions for wording, based on terms from PPPs where the public authority makes a contribution to the contract/project.

Reply:
I think the charges extension is probably the best mapping for this field too, although the description is vague. Do we have access to any more detail on the definition?

Reply:
The 2014/23/EU Directive might have an article or clause using similar words to the form element - I’m in transit otherwise I would check myself.

Reply:
We could call it `prizesAndPayements` or `financialAdvantages`. In the directive, prizes and payments always go together. And it could be added to the contract's `financeSummary` ([schema documentation](https://extensions.open-contracting.org/en/extensions/ppp/master/schema/#financesummary)).

What do you think?

Reply:
I think use the charges extension as Duncan suggested, setting `paidBy` to 'government' and `title` to the translation of "Prizes, payments or other financial advantages provided by the contracting authority/entity".

Reply:
@ColinMaudry Can you update the mapping for `VAL_PRICE_PAYMENT` to be similar to the mapping for `VAL_REVENUE`, with the differences noted in my last comment?

Reply:
Done in https://github.com/open-contracting/european-union-support/pull/28/commits/25ccd07dc76b7420bf260be6755837c30d3edb5a